src/test/java/Class3/Homework2.java

```diff
@@ -0,0 +1,153 @@
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://www.facebook.com/");
+        driver.findElement(By.name("login")).click();
+        try {
+            Thread.sleep(2000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        String actualText = driver.findElement(By.xpath("//div[text()='The email or mobile number you entered isn’t connected to an account. ']")).getText();
+
+        String expectedText = "The email or mobile number you entered isn’t connected to an account. Find your account and log in.";
+        Assert.assertEquals(actualText, expectedText, "Expected text is not matching.");
+        driver.quit();
+    }
+
+
+
+
+
+    /**
+     *  TC-2: Verify Gender error msg on Create New Account screen
+     *  1. Launch facebook.com
+     *  2. Click the Create New Account button
+     *  3. Enter firstName
+     *  4. Enter lastName
+     *  5. Enter mobile number
+     *  6. Enter new password
+     *  7. Click the "Sign Up" button
+     *  8. verify "Please choose a gender. You can change who can see this later."
+     *      error msg is displayed
+     */
+
+    @Test
+    public void verifyGenderErrorMsg(){
+
+        System.setProperty("webdriver.chrome.driver", "Drivers/chromedriver");
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://www.facebook.com/");
+
+        try {
+            Thread.sleep(2000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+
+        // Click the Create New Account button
+        String createNewAccountXpath = "//*[text()='text']";
+        By NewAccountLocator = By.linkText(createNewAccountXpath);
+        WebElement createNewAcct = driver.findElement(NewAccountLocator);
+        createNewAcct.click();
+
+        //Enter firstName
+        String FirstNameXpath = "//input[contains(@name,'first')]";
+        By FirstNameLocator = By.xpath(FirstNameXpath);
+        WebElement FirstNameBox = driver.findElement(FirstNameLocator);
+        FirstNameBox.sendKeys("Taaaaa");
+
+        /* short way
+        By FirstNameLocator = By.xpath("//input[contains(@name,'first')]");
+        WebElement FirstNameBox = driver.findElement(FirstNameLocator);
+        FirstNameBox.sendKeys("Taaaaa");
+
+        or
+
+        driver.findElement(By.xpath("//input[contains(@name,'first')]")).sendKeys("Taaaaa");
+         */
+
+        // Enter lastName
+        driver.findElement(By.xpath("//input[contains(@name,'last')]")).sendKeys("Haaaa");
+
+        //Enter mobile number
+        driver.findElement(By.xpath("//*[text()='Mobile number or email']")).sendKeys("999 999 9999");
+
+        //Enter new password
+        driver.findElement(By.xpath("//*[text()='New password']")).sendKeys("abc@123");
+
+        //Click the "Sign Up" button
+        driver.findElement(By.xpath("//*[text()='Sign Up' and @xpath='1']")).click();
```

Comment:
never use "xpath attribute in xpath....as it is there by Chropath to indicate the line of DOM with entered xpath

Reply:
Fixed